### PR TITLE
Report update

### PR DESCRIPTION
**Pull request by sjshoemak**

### CODE REVIEW

**Comment by AaronHampshire** [comment]
Bootstrap tabs are not responsive, so when the window is small, the tabs stack up on each other. I increased the width and hid the overflow. It's not nice, but it works.

**Comment by AaronHampshire** [comment]
Thumbs up from me to merge.

**Comment by cdschmitz** [comment]
I looked at the report for a couple with no houses and no categories.  It looked ridiculous with lots of blank space and no data.  Added a check to ReportView that requires at least one House and one Category to display the report.  Otherwise it shows a view which communicates to the user what they need to do before a report can be generated.

File: RealEstate/apps/core/static/style.css [hunk 1]
```diff
@@ -251,3 +245,105 @@ a.mail {
...
+#fork {
+    position: absolute;
+    top: 0;
+    right: 0;
+    border: 0;
+}
+
+.legend {
+    width: 10em;
+    border: 1px solid black;
```
Comment (cdschmitz) [inline RealEstate/apps/core/static/style.css:314]: I added a new couple with just the default categories and one house.  This border did not expand to fit around all of the content (for the Weighting tab).  It looked fine without it.  I think we can remove it

**Comment by cdschmitz** [comment]
Now that I type it out maybe it should be minimum 2 houses otherwise whats the point?  I guess I will leave it at one for now

**Comment by cdschmitz** [comment]
Some styling issues but merge-able I think

**Pull request merged**